src/Features/CSharp/Portable/AddImport/CSharpAddImportFeatureService.cs

```diff
@@ -374,8 +375,9 @@ private string GetUsingDirectiveString(INamespaceOrTypeSymbol namespaceOrTypeSym
                 }
 
                 var addImportService = document.GetLanguageService<IAddImportsService>();
+                var generator = SyntaxGenerator.GetGenerator(document);
```

Comment:
I will attempt to move the changes to add imports service and unnecessary imports service to a separate PR.